x/ccv/provider/keeper/proposal_test.go

```diff
@@ -569,48 +572,12 @@ func TestStopConsumerChain(t *testing.T) {
 			require.NoError(t, err)
 		}
 
-		testProviderStateIsCleaned(t, ctx, providerKeeper, "chainID", "channelID")
+		testkeeper.TestProviderStateIsCleanedAfterConsumerChainIsStopped(t, ctx, providerKeeper, "chainID", "channelID")
```

Comment:
Because this is used in `relay_test.go` as well now, the function is moved to `testutil/keeper/unit_test_helpers.go`.